Files/ViewModels/SettingsViewModel.cs

```diff
@@ -536,6 +536,23 @@ public bool PinRecycleBinToSideBar
             }
         }
 
+        ///// <summary>
+        ///// Gets or sets the value indicating whether the preview pane should be shown.
+        ///// </summary>
+        //public bool PreviewPaneEnabled
```

Comment:
unnecessary code?

Reply:
Yeah, I forgot to delete that after moving the setting to InstanceViewModel.